Source/Details/ASGraphicsContext.mm

```diff
@@ -121,7 +121,9 @@ NS_INLINE void ASConfigureExtendedRange(UIGraphicsImageRendererFormat *format)
   }
 
   // Bad OS or experiment flag. Use UIGraphics* API.
-  UIGraphicsBeginImageContextWithOptions(size, opaque, scale);
+  if (size.width > 0 && size.height > 0) {
+    UIGraphicsBeginImageContextWithOptions(size, opaque, scale);
```

Comment:
If we have an invalid size, I think we should exit early and return nil. Shortly after starting the image context we call `UIGraphicsGetImageFromCurrentImageContext` and `UIGraphicsEndImageContext`. I'm not sure what the behavior is if we end a graphics context without starting one.

Reply:
I've switched to an early return